test/export/test_converter.py

```diff
@@ -8,18 +8,47 @@
 
 
 class TestConverter(TestCase):
+    def _check_equal_ts_ep_converter(self, mod_func, inp):
+        mod = mod_func()
+        ts_model = torch.jit.script(mod)
+        ep = TS2EPConverter(ts_model, inp).convert()
+        torch.testing.assert_close(ep.module()(*inp)[0], mod(*inp))
```

Comment:
the hardcoded `[0]` here is a hack, and only work for model with single output. 

the proper way to handle multiple outputs is to first pytree.flatten the output, and compare the values in a loop.